src/tool/core/helpers/problem_code.rs

```diff
@@ -265,6 +275,7 @@ impl FunctionArgType {
                 Self::does_pass_basic_vec_tests(line)?;
                 format!("TreeRoot::from(\"{line}\").into()")
             }
+            FunctionArgType::FATOther { raw: _ } => line.to_string(), // Assume input is fine and pass on verbatim,
```

Comment:
Why would that be better? Right now it will just pass through the input and in a lot of cases that will just work. Like in the case of i64 which I used for testing then it produces the correct output without doing anything special but if we add a `todo!()` it will cause the user to need to change it before they can use it.